src/hooks/useSample.ts

```diff
@@ -0,0 +1,9 @@
+import useSWR from 'swr'
+
+import { fetcher } from '../lib/api/fetch'
+
+const key = 'https://api.hnpwa.com/v0/ask.json'
+
+export default function useSampleFetch() {
+  return useSWR<any>(key, fetcher)
+}
```

Comment:
이런 상황일 때는 `any`보다는 제네릭타입을 사용하는 것이 더 단단한 타입설계에 도움이 될 것 같습니다!

```suggestion
export default function useSampleFetch<T>() {
  return useSWR<T>(key, fetcher)
}
```